charts/tp-cp-core/charts/identity-management/templates/idm.yaml

```diff
@@ -212,21 +212,20 @@ spec:
         - name: IDENTITY_MANAGEMENT_OPTS
           value: -DrootLogLevel=DEBUG
         - name: IDENTITY_MANAGEMENT_JWT_KEYSTORE_URL
-          value: {{ .Values.global.tibco.idm_cp_jwt_keystore_url }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ include "identity-management-jwt-keystore-url" . }}
+              key: IDENTITY_MANAGEMENT_JWT_KEYSTORE_URL
```

Comment:
this is not working. See https://tibco.slack.com/archives/C07LPB1RC04/p1740439811269109?thread_ts=1740378929.673809&cid=C07LPB1RC04